File: Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/preprocessor/CodeDefinitionInfo.java
package org.cqframework.cql.cql2elm.preprocessor;

import org.cqframework.cql.gen.cqlParser;

/**
 * Created by Bryn on 5/22/2016.
 */
public class CodeDefinitionInfo {
    private String name;
    private cqlParser.CodeDefinitionContext definition;

    public String getName() {
        return name;
    }

    public void setName(String value) {
        name = value;
    }

    public cqlParser.CodeDefinitionContext getDefinition() {
        return definition;
    }

    public void setDefinition(cqlParser.CodeDefinitionContext value) {
        definition = value;
    }

    public CodeDefinitionInfo withName(String value) {
        setName(value);
        return this;
    }

    public CodeDefinitionInfo withDefinition(cqlParser.CodeDefinitionContext value) {
        setDefinition(value);
        return this;
    }
}

File: Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/preprocessor/CodeDefinitionInfo.kt
package org.cqframework.cql.cql2elm.preprocessor

import org.cqframework.cql.gen.cqlParser.CodeDefinitionContext

/** Created by Bryn on 5/22/2016. */
class CodeDefinitionInfo(val name: String, override val definition: CodeDefinitionContext) :
    BaseInfo(definition)
